src/languages/es.js

```diff
@@ -820,6 +820,17 @@ export default {
     },
     emojiPicker: {
         skinTonePickerLabel: 'Elige el tono de piel por defecto',
+        headers: {
+            frequentlyUsed: 'Usado frecuentemente',
+            smileysAndPeople: 'Emoticonos y personas',
+            animalsAndNature: 'Animales y naturaleza',
+            foodAndDrinks: 'Alimentos y bebidas',
+            travelAndPlaces: 'Viajes y lugares',
+            activities: 'Actividades',
+            objects: 'Objetos',
+            symbols: 'Simbolos',
```

Comment:
```suggestion
            symbols: 'Símbolos',
```

Reply:
Is it me? I see no change here.
![image](https://user-images.githubusercontent.com/24370807/148999325-12f6276e-e3b5-464d-8d31-e3ce5f508970.png)

Reply:
aha its `i`